### PR TITLE
More styling

### PR DESCRIPTION
**Pull request by gminaya**
**On this PR:**
1) Changed some CSS to SCSS
2) Chanced NewLocationForm from a Modal to a Drawer for design consistency
3) Moved some inline styling to its own files

### CODE REVIEW

File: src/styles/components/locationCard.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import 'styles/helpers/colors.scss';
+
+.location-card {
+  flex: 0 1 auto;
```
Comment (amhed) [inline src/styles/components/locationCard.scss:4]: nice! short form

File: src/styles/components/newLocationForm.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.form-container {
```
Comment (amhed) [inline src/styles/components/newLocationForm.scss:1]: .form-container probably repeats in multiple places. Should you pull up?

**Review by amhed — APPROVED** [review]

File: src/styles/components/newLocationForm.scss (hunk 2 above)
Comment (gminaya) [reply src/styles/components/newLocationForm.scss]: I changed the class name to 'new-location-form-container' since each form has a different styling

Comment (amhed) [reply src/styles/components/newLocationForm.scss]: Why does each form have different styling? form styling should be consistent throughouth the app

**Pull request merged**